admin/css/picker.css

```diff
@@ -1,22 +1,26 @@
 /* Todo: Change to scss, separate styles that are essential to the ImagePicker component to work (if there are any)
      from ArchivePicker styles*/
 
-.archive-picker-search input[type=submit] {
-  display: inline-block;
+.archive-picker-search {
+  margin-top: 20px;
+}
+
+.archive-picker-search input {
+  margin-right: 10px;
 }
 
 .picker-list {
   max-width: 60%;
   max-height: 75vh;
   overflow-y: scroll;
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  grid-template-columns: calc(33% - 10px) calc(33% - 10px) calc(33% - 10px);
```

Comment:
Minor tweak here, to make it look better on mobile, we could use something similar to the [Gallery Block](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/blob/master/assets/src/styles/blocks/Gallery_grid.scss#L32-L46).

For small screens:
```
grid-template-columns: repeat(2, 1fr);
```

For bigger screens:
```
grid-template-columns: repeat(3, 1fr);
```